components/automate-gateway/api/iam/v2/policy.proto

```diff
@@ -129,7 +231,15 @@ service Policies {
     option (chef.automate.api.policy).action = "delete";
     option (chef.automate.api.iam.policy).resource = "iam:policies:{id}:members";
     option (chef.automate.api.iam.policy).action = "iam:policyMembers:delete";
-    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+			tags: "policies";
+		};
+  };
+  /*
+  Add policy members
+
+  Add specific members to the member list for a specific policy.
```

Comment:
Add after this:

"Ensure each element of the `members` array is in the correct [Member Expression]({{< relref "iam-v2-guide.md#member-expressions" >}}) format."